sjsonnet/test/src-jvm/sjsonnet/ErrorTestsJvmOnly.scala

```diff
@@ -0,0 +1,17 @@
+package sjsonnet
+
+import utest._
+
+object ErrorTestsJvmOnly extends TestSuite with ErrorTestsBase{
+  val tests = Tests{
+    test("array_recursive_manifest") - check(
```

Comment:
StackOverflow errors can't be `catch`ed in Scala Native and they crash the process.
That's why I moved them to be run only in the JVM subproject.